features/gardener/file.include/usr/lib/tmpfiles.d/gardener.conf

```diff
@@ -0,0 +1,5 @@
+# create directories on /var that are not created automatically
+d /var/lib/gardener-gardenlinux 0755 root root
+C /var/lib/gardener-gardenlinux/01_configure_lsm.sh 0755 root root
+C /var/lib/gardener-gardenlinux/02_iptables_backend.sh 0755 root root
+C /var/lib/gardener-gardenlinux/restart_kernel.sh 0755 root root
```

Comment:
Please leave this file out, they are no longer part of Garden Linux and no longer required by Gardener.